### ISSUE TITLE
[Terminal > Command Palette]: No hint is given to visual/non-visual user about the jumping between command line mode or Actions Search mode.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12042]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Command Palette using 'Ctrl + Shift + p'.
3. Press 'Backspace' key.
4. Verify that hint is given to visual/non-visual user about the jumping between command line mode or Actions Search mode or not.

**User Experience:**
Users UX will not be good if user will not know how use certain functionality.

**Attachments:**
[No hint is given to visualnon-visual user about the jumping between command line mode or Actions Search mode..zip](https://github.com/microsoft/terminal/files/7779299/No.hint.is.given.to.visualnon-visual.user.about.the.jumping.between.command.line.mode.or.Actions.Search.mode.zip)

### Expected Behavior

Hint should be given to visual/non-visual user about how they can jump between command line mode or Actions Search mode inside Command Palette Window.

### Actual Behavior

No hint is given to visual/non-visual users about how they can jump between command line mode or Actions Search mode inside Command Palette Window. 
Visual Placeholder Lable is given as 'Type a command name' and 'Enter a wt command line to run'.
Screen Reader announce, 'Command line mode' and 'Actions Search mode' when we jump between two modes but no hint about how they can jump between these two modes. Also Screen Reader remains silent when focus goes on '>' icon.

Observation: Same issue repro with NVDA and JAWS Screen Reader.
